dallinger/command_line/docker_ssh.py

```diff
@@ -145,6 +147,28 @@ def prepare_server(host, user):
         print("Docker compose already installed")
 
 
+def copy_docker_config(host, user):
+    executor = Executor(host, user)
+
+    local_docker_conf_path = os.path.expanduser("~/.docker/config.json")
+    if os.path.exists(local_docker_conf_path):
+        with open(local_docker_conf_path, "rb") as fh:
+            local_file_contents = fh.read()
+        remote_has_conf = executor.run(
+            "ls ~/.docker/config.json > /dev/null && echo true || true"
+        ).strip()
+        if remote_has_conf == "true":
+            remote_sha, _ = executor.run("sha256sum ~/.docker/config.json").split()
+            local_sha = hashlib.sha256(local_file_contents).hexdigest()
+            if local_sha != remote_sha:
+                # Move the remote file to a temporary location
+                executor.run(
+                    "mv ~/.docker/config.json  ~/.docker/config.json.$(date +%d-%m-%Y-%H:%M.bak)"
```

Comment:
Any danger that I run the command twice in the same minute and overwrite my backup file irretrievably?

Reply:
If you run it twice in the same minute the first run will copy your local file, so your second run should leave it as is (overwriting it with the same contents).